.bazelrc

```diff
@@ -242,7 +242,7 @@ build:windows --override_repository="boringssl=C:\\opt\\boringSSL-SwiftPM"
 
 # Windows config default flags
 build:windows --define=CLOUD_DISABLE=1
-build:windows --define=PYTHON_DISABLE=0
+build:windows --define=PYTHON_DISABLE=1
```

Comment:
This also disables python_nodes. We do not want it on windows ? Also we do not test python compilation in this settings on windows.